src/vivarium_public_health/risks/data_transformations.py

```diff
@@ -417,6 +417,45 @@ def validate_distribution_data_source(builder, risk: EntityString):
     else:
         raise ValueError(f"Unknown risk type {risk.type} for risk {risk.name}")
 
+    validate_lookup_configuration(builder, risk)
+
+
+def validate_lookup_configuration(builder, risk: EntityString) -> None:
```

Comment:
Where is this being called?

Reply:
This method has been deleted based on our discussion today.